spec/features/batch_publishing_spec.rb

```diff
@@ -95,6 +95,26 @@ def expect_message(attributes, model_name = "TestUser", routing_key = "TestUser"
       end
     end
 
+    # NOTE: for compatibility with active_job 4.2.11
+    context "attributes with symbolic values in hashes" do
+      let(:attributes) do
+        [
+          half_bad: { bad_inside: { foo: :bar }, good_inside: { foo: "bar" } },
+        ]
+      end
+
+      it "filters these values out" do
+        publisher.publish
+        job = ActiveJob::Base.queue_adapter.enqueued_jobs.last
+        params = job[:args][1]
+        expect(params.first.keys.size).to eq(2)
+        expect(params.first["half_bad"]).to include(
+          "bad_inside" => { "_aj_symbol_keys" => [] },
```

Comment:
Do we really want to exclude symbols rather than convert them to strings? 🤔

Reply:
In previous versions it excluded symbolic keys
```
  def filter_safe_for_serialization(hash)
    hash.select { |_, value| value.is_a?(String) || value.is_a?(Numeric) }.to_h
  end
```